lib/high_voltage/page_finder.rb

```diff
@@ -32,7 +32,7 @@ def clean_path
 
     def clean_id
       @page_id.tr("^#{VALID_CHARACTERS}", "").tap do |id|
-        if id.blank?
+        if id.blank? || id.first == "."
```

Comment:
Since this is now getting more complex, how about extracting a private method called `valid_page_id?` Then this could be:

```ruby
unless valid_page_id?(id) do
  raise InvalidPageIdError.new "Invalid page id: #{@page_id}"
end
```